Assets/HoloToolkit/Input/README.md

```diff
@@ -13,71 +39,106 @@ Torus shaped CursorOnHolograms when user is gazing at holograms and point light
 #### CursorWithFeedback.prefab
 Torus shaped cursor that follows the user's gaze and HandDetectedFeedback asset to give feedback to user when their hand is detected in the ready state.
 
-#### FocusedObjectKeywordManager.prefab
-Keyword manager pre-wired to send messages to object being currently focused via FocusedObjectMessageSender component.
-You can simply drop this into your scene and be able to send arbitrary messages to currently focused object.
+#### DefaultCursor.prefab
+3D animated cursor that follows the user's gaze and uses the Unity animation system to handle its various states. This cursor imitates the HoloLens Shell cursor.
 
-#### SelectedObjectKeywordManager.prefab
-Keyword manager pre-wired to send messages to object being currently selected via SelectedObjectMessageSender comoponent.
-You can simply drop this into your scene and be able to send arbitrary messages to currently selected object.
+#### HoloLensCamera.prefab
```

Comment:
It's much more powerful as far as customization goes, and it leverages the same scripts already used for simulating hand movement in the editor. This change remove the "Main Camera.prefab", as this ones fully replaces it.